bin/build-static-image.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+if [ -f ".env.static.local" ]
+then
+  source .env.static.local
```

Comment:
Add a comment that it isn't a regular .env file, it needs `export KEY=value` instead